### ISSUE TITLE
Target framework of nuget package

### ISSUE THREAD

**Issue by runceel** [kenakamu/clova-cek-sdk-csharp#2]
Hi @kenakamu ,

I'm trying CEK.CSharp on Azure Functions v2 runtime.
When I added this library using `dotnet add package` command, following message would be occurred.

```
warn : パッケージ 'CEK.CSharp 0.1.4' はプロジェクトのターゲット フレームワーク '.NETCoreApp,Version=v2.1' ではなく '.NETFramework,Version=v4.6.1' を使用して復元されました。
このパッケージは、使用しているプロジェクトとの完全な互換性がない可能性があります。
```
On your NuGet update log, there are `0.1.3 - Convert to .NET Standard 1.3`. 
However, in version 0.1.4, the target framework is .NET Framework v4.6.1.

I want to use the library on the azure functions v2(.NET Core 2.1).
Could you check the target version?

**Comment by kenakamu** [comment]
This happed as I added dependency references. I could run it on Azure Function v2.

```csharp

#r "Newtonsoft.Json"

using System.IO;
using Microsoft.AspNetCore.Mvc;
using Microsoft.Azure.WebJobs;
using Microsoft.AspNetCore.Http;
using Microsoft.Azure.WebJobs.Host;
using Newtonsoft.Json;
using Microsoft.Extensions.Logging;
using CEK.CSharp;
using System.Threading.Tasks;
using CEK.CSharp.Models;

public static IActionResult Run(HttpRequest req, ILogger log)
{
    log.LogInformation("C# HTTP trigger function processed a request.");
            ClovaClient client  = new ClovaClient();
            var request = client.GetRequest(req.Headers["SignatureCEK"], req.Body, true).Result;
            var response = new CEKResponse();
             // get session information by specifying default value in case no session information
            var mySessionValue = request.GetSessionAttribute("mySessionKey", "defaultValue");
            // adding session information
            response.AddSession("mySessionKey", "mySessionValue");

            switch (request.Request.Type)
            {
                case RequestType.LaunchRequest:
                    // Single Text Reply
                    response.AddText("Welcome to CEK", Lang.En);
                    response.ShouldEndSession = false;
                    break;
                case RequestType.SessionEndedRequest:
                    response.AddText("Good bye!", Lang.En);
                    response.ShouldEndSession = true;
                    break;
                case RequestType.IntentRequest:
                    switch (request.Request.Intent.Name)
                    {
                        case "Clova.YesIntent":
                            // Add single URL Response and Text Reprompt
                            response.AddUrl("https://clova-common.line-scdn.net/dice/rolling_dice_sound.mp3");
                            response.AddRepromptText("Tell me something, please", Lang.En);                            
                            response.ShouldEndSession = false;
                            break;
                        case "Clova.NoIntent":
                            // Add Brief and Verbose as SpeechSet
                            response.AddBriefText("Brief explain.", Lang.En);
                            response.AddVerboseText("Detail explain 1.", Lang.En);
                            response.AddVerboseText("Detail explain 2.", Lang.En);
                            response.AddVerboseUrl("https://clova-common.line-scdn.net/dice/rolling_dice_sound.mp3");
                            response.ShouldEndSession = false;
                            break;
                        case "Clova.GuideIntent":
                            // Add multiple Reposonses and Reprompts
                            response.AddText("Sure!", Lang.En);
                            response.AddUrl("https://clova-common.line-scdn.net/dice/rolling_dice_sound.mp3");
                            response.AddText("Let me explain how to use it!", Lang.En);
                            response.AddRepromptText("Did you understand?", Lang.En);
                            response.AddRepromptText("Now tell me what you want.", Lang.En);
                            response.AddRepromptUrl("https://clova-common.line-scdn.net/dice/rolling_dice_sound.mp3");
                            response.ShouldEndSession = false;
                            break;
                    }
                    break;
            }
            return new OkObjectResult(response);
        }
```

**Comment by kenakamu** [comment]
I just realized that my package target was wrong. I fix it and release it as 0.1.5.

**Comment by runceel** [comment]
Thank you!!
I'll use this library.
